packages/desktop-gui/src/specs/specs-list.jsx

```diff
@@ -156,6 +165,28 @@ class SpecsList extends Component {
     return spec.hasChildren ? this._folderContent(spec, nestingLevel) : this._specContent(spec, nestingLevel)
   }
 
+  _setFocus = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
+
+  _setBlur = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
```

Comment:
Can combine redundant function into one action.

```suggestion
  _toggleFocus = () => {
    this.setState({
      isFocused: !this.state.isFocused,
    })
  }
```

Reply:
@jennifer-shehan yes! I didn't use `DRY` principles here. I have addressed the comment & pushed the changes.